open-api-spec.yaml

```diff
@@ -12,10 +12,11 @@ info:
 
     All resource and collection resource representations use JSON and the media-type `application/json`.  
 
-    The `account` resource has two collections of sub-resources and a single **account holder** sub-resource:   
+    The `account` resource has three collections of sub-resources and a single **account holder** sub-resource:   
     1. `/incoming-payments` contains the **incoming payment** sub-resources  
     2. `/outgoing-payments` contains the **outgoing payment** sub-resources  
-    3. `/holder` contains the **outgoing payment** sub-resources  
+    3. `/quotes` contains the **quote** sub-resources  
+    4. `/holder` contains the **outgoing payment** sub-resources  
```

Comment:
Do you want to remove the holder resource in your PR as well or should I create a new one for that?

Reply:
I can remove it.